sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DescribeNamespaceExec.scala

```diff
@@ -44,10 +44,13 @@ case class DescribeNamespaceExec(
     }
 
     if (isExtended) {
-      val properties = metadata.asScala -- CatalogV2Util.NAMESPACE_RESERVED_PROPERTIES
-      if (properties.nonEmpty) {
-        rows += toCatalystRow("Properties", properties.toSeq.mkString("(", ",", ")"))
+      val properties = metadata.asScala.toMap -- CatalogV2Util.NAMESPACE_RESERVED_PROPERTIES
```

Comment:
Adding `toMap` similar to https://github.com/apache/spark/blob/8238cdd443d5bc8557f7e0e76f40c66bf8715c5a/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2SessionCatalog.scala#L321.

The Scala Map implementation uses a list underneath up to 4 elements and maintains it in the order it was inserted. So this make tests such as https://github.com/apache/spark/blob/8238cdd443d5bc8557f7e0e76f40c66bf8715c5a/sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala#L778 pass.

Maybe, just make the tests more robust?

Reply:
I just changed the command to sort the properties by keys (similar to `show tblproperties`).